build/marionette.state.js

```diff
@@ -35,7 +39,7 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
     //   preventDestroy: {boolean} If true, then this will not destroy on `component` destroy.
     // }
     constructor: function constructor() {
-      var _ref = arguments[0] === undefined ? {} : arguments[0];
+      var _ref = arguments.length <= 0 || arguments[0] === undefined ? {} : arguments[0];
```

Comment:
It looks like maybe you had a different version of Babel when you ran the build, which is possible.  Just `git reset HEAD~1 --soft`, make a new commit without this, and force push to your branch.